<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QScrollArea" name="scroll_area"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>905</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="action_New"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="actionSave_All"/>
    <addaction name="actionSave_Image"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Segment">
    <property name="title">
     <string>Segment</string>
    </property>
    <addaction name="actionLazyBrush"/>
    <addaction name="actionGrabCut"/>
    <addaction name="actionHist"/>
    <addaction name="actionMixture"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <widget class="QMenu" name="menuImageWindow">
     <property name="title">
      <string>ImageWindow</string>
     </property>
     <addaction name="actionHorizontal"/>
     <addaction name="actionVertical"/>
    </widget>
    <addaction name="menuImageWindow"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Segment"/>
   <addaction name="menu_Window"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100000</width>
     <height>524287</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Segment Param</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QStackedWidget" name="dockWidgetContents"/>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="text">
    <string>Save All</string>
   </property>
  </action>
  <action name="actionSave_Image">
   <property name="text">
    <string>Save Image</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionLazyBrush">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>LazyBrush</string>
   </property>
  </action>
  <action name="actionGrabCut">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>GrabCut</string>
   </property>
  </action>
  <action name="actionHist">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>GraphCut</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionMixture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mixture</string>
   </property>
  </action>
  <action name="actionHorizontal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Horizontal</string>
   </property>
  </action>
  <action name="actionVertical">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Vertical</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
